src/main/webapp/app/overview/course-conversations/layout/conversation-selection-sidebar/conversation-selection-sidebar.component.ts

```diff
@@ -43,6 +43,10 @@ export class ConversationSelectionSidebarComponent implements AfterViewInit, OnI
     displayedStarredConversations: ConversationDto[] = [];
     channelConversations: ChannelDTO[] = [];
     displayedChannelConversations: ChannelDTO[] = [];
+    exerciseChannelConversations: ChannelDTO[] = [];
```

Comment:
Maybe a Map (ChannelSubType -> Channel[]) would work here? This might help reduce the basically duplicated lines below.